src/main/resources/assets/pyrotech/patchouli_books/book/zh_cn/entries/bloom/tools_redstone.json

```diff
@@ -5,7 +5,7 @@
   "pages": [
     {
       "type": "text",
-      "text": "$(info) $(thing)红石工具$()会在耐久消耗或待在$(thing)红石矿石$()附近时被激活。$(p)$(info) $(thing)红石工具$()会在玩家待在$(thing)红石矿石$()附近时缓慢回复耐久。$(p)$(info) Active $(thing)Redstone Tools$() resist damage and are more likely to remain active by taking damage."
+      "text": "$(info) $(thing)红石工具$()会在耐久消耗或待在$(thing)红石矿石$()附近时被激活。$(p)$(info) $(thing)红石工具$()会在玩家待在$(thing)红石矿石$()附近时缓慢回复耐久。$(p)$(info) 激活的$(thing)红石工具$() resist damage and are more likely to remain active by taking damage."
```

Comment:
What does "resist damage" means?
Won't the tools take damage? Or just take damage hardly?

Reply:
It means that there is less of a chance that they will take damage when used.

Reply:
okay, thanks